pymongo/operations.py

```diff
@@ -773,6 +773,19 @@ def document(self) -> dict[str, Any]:
         """
         return self.__document
 
+    def __repr__(self) -> str:
```

Comment:
ruff is still failing because it wants you to use f-strings. Do you have pre-commit set up?

Reply:
Sorry, forgot my `pre-commit install`! I think I just need to add a more meaningful test to the `SearchIndexModel` test class to finish this.